### PR TITLE
Fix stretched images in community posts

### LINKED ISSUE #4852: [Bug]: Images strechted on Community page
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Go to https://www.youtube.com/@ludwig/community
2. See that some posts are stretched
 

https://github.com/FreeTubeApp/FreeTube/assets/73130443/469f3dd5-38c5-41fa-836e-352694b41c5d

### Expected Behavior

Community posts shouldn't be stretched.

### Issue Labels

visual bug

### FreeTube Version

v0.20.0-nightly-4137 Beta

### Operating System Version

Win10 22H2

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

_No response_

### Additional Information

_No response_

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

### PR DESCRIPTION
**Pull request by tzwel**
# Fix stretched images in community posts

## Pull Request Type

- [x] Bugfix
- [ ] Feature Implementation
- [ ] Documentation
- [ ] Other

## Related issue
closes #4852

## Description

I added `object-fit: cover;` to images so they crop in the same way YouTube images do. Fixed a couple of heights and widths to make this possible. Not exactly related, but added a little bit more of margin between posts and reactions to make them a bit more legible.

## Screenshots 
Before:
image

After:
image

## Testing
The code changes are very small and don't need to be tested VERY throughly, but I inspected the changes on many community posts and it all works as intended
 

## Desktop

- **OS:** Windows
- **OS Version:** 2004
- **FreeTube version:** v0.21.1 Beta

### CODE REVIEW

**Review by tzwel — COMMENTED** [review]
Fix stretched images in community posts

File: src/renderer/components/ft-community-post/ft-community-post.scss [hunk 1]
```diff
@@ -21,8 +21,14 @@
...
-  block-size: 100%;
-  inline-size: 100%;
+    inline-size: 100%;
+    block-size: 640px;
+    object-fit: cover;
+}
+
+.swiperSlide {
+  max-block-size: 640px;
+  max-inline-size: 640px;
```
Comment (PikachuEXE) [inline src/renderer/components/ft-community-post/ft-community-post.scss:31]: not sure if a fixed width is desired here (I got no idea how community post UI is designed, pending comments from others

**Review by PikachuEXE — CHANGES_REQUESTED** [review]

File: src/renderer/components/ft-community-post/ft-community-post.scss (hunk 1 above)
Comment (tzwel) [reply src/renderer/components/ft-community-post/ft-community-post.scss:31]: every youtube community post displays always at both fixed width and height (at least from my testing)

File: src/renderer/components/ft-community-post/ft-community-post.scss [hunk 2]
```diff
@@ -21,8 +21,14 @@
 }
 
 .communityImage {
-  block-size: 100%;
-  inline-size: 100%;
+    inline-size: 100%;
+    block-size: 640px;
+    object-fit: cover;
```
[the remark this answers is not in the document]
Comment (tzwel) [reply src/renderer/components/ft-community-post/ft-community-post.scss]: fixed

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — COMMENTED** [review]
Before

https://github.com/user-attachments/assets/e7e0defa-d307-4e4c-8e60-31de76d22a2e

After

https://github.com/user-attachments/assets/30db05fa-61d4-4ebb-9b90-74d3f18d233e

**Comment by tzwel** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc the way they display now is actually exactly how YouTube displays the images (you can check). Maybe I should add some click-to-expand functionality?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
My bad didn't check how it looked like on YT end. 
I don't think its necessary to expand the functionality in this PR we can leave that for a separate one.

Did you close the PR by mistake?

**Comment by tzwel** [comment]
Forgot to create a branch for my changes, then completely f-ed up the git history and deleted my fork enraged not thinking about the consequences, rookie mistake. I accidentally committed some other changes not meant for this PR. Can open a new PR if you want to merge/request changes for this

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No worries. Please create a new PR if you still want to work on it

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @tzwel would you still be open to creating a new PR?

**Comment by tzwel** [comment]
Hey, life has gotten in the way and I kinda abandoned my efforts in the project. But why not make the PR yourself? All my commits are there, and they probably should still work (although I'm not sure, I haven't updated for ages)

@efb4f5ff-1298-471a-8973-3d47447115dc

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@caetano-dev is this something you would be interested in to pick up?

**Pull request closed without merging**